### PR TITLE
Nerfs the blueshield

### PR DESCRIPTION
**Pull request by alexkar598**
:cl: alexkar598
del: Removed the detective gun from the blueshield.
tweak:The blueshield Aegis SG7 Laser Gun have seen a rework,it can now fire 15 disabler shot and 7 lethal shot.
balance: The blueshield access has been significantly reduced.
/:cl:

old access:
```
ACCESS_SECURITY, ACCESS_SEC_DOORS, ACCESS_COURT,
ACCESS_WEAPONS,ACCESS_MEDICAL, ACCESS_ENGINE, 
ACCESS_AI_UPLOAD, ACCESS_EVA,
ACCESS_ALL_PERSONAL_LOCKERS, ACCESS_MAINT_TUNNELS, 
ACCESS_BAR, ACCESS_JANITOR, ACCESS_CONSTRUCTION, ACCESS_MORGUE,
ACCESS_CREMATORIUM, ACCESS_KITCHEN, ACCESS_CARGO, ACCESS_CARGO_BOT,
ACCESS_MAILSORTING, ACCESS_QM, ACCESS_HYDROPONICS, ACCESS_LAWYER,
ACCESS_THEATRE, ACCESS_CHAPEL_OFFICE, ACCESS_LIBRARY, 
ACCESS_RESEARCH, ACCESS_MINING, ACCESS_VAULT, ACCESS_MINING_STATION,
ACCESS_HOP, ACCESS_RC_ANNOUNCE, ACCESS_KEYCARD_AUTH, ACCESS_GATEWAY, 
ACCESS_MINERAL_STOREROOM, ACCESS_BLUESHIELD, 
ACCESS_CAPTAIN, ACCESS_RD, ACCESS_HOS, ACCESS_CMO, ACCESS_CE
```

new access:
```
ACCESS_SECURITY,ACCESS_MEDICAL,ACCESS_MORGUE,ACCESS_ENGINE,
ACCESS_MAINT_TUNNELS,ACCESS_TELEPORTER,ACCESS_EVA,ACCESS_HEADS,
ACCESS_ALL_PERSONAL_LOCKERS,ACCESS_COURT,ACCESS_RESEARCH,
ACCESS_CONSTRUCTION,ACCESS_CARGO,ACCESS_SEC_DOORS,ACCESS_WEAPONS,
ACCESS_RC_ANNOUNCE,ACCESS_KEYCARD_AUTH,ACCESS_BLUESHIELD,ACCESS_ATMOSPHERICS
```

### CODE REVIEW

**Comment by alexkar598** [comment]
gun
gun pic btw

File: sunset/code/game/objects/items/blueshield.dm [hunk 1]
```diff
@@ -34,14 +33,6 @@
 /obj/item/storage/box/blueshield/laser/PopulateContents()
 	new /obj/item/gun/energy/e_gun/blueshield(src)
 
-/obj/item/storage/box/blueshield/revolver //go and die please
```
Comment (alexkar598) [inline sunset/code/game/objects/items/blueshield.dm]: told you to die,now i kill you

File: sunset/code/modules/projectiles/gun/blueshield.dm [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 	name = "advanced stun revolver"
-	desc = "An advanced stun revolver with the capacity to shoot both electrodes and lasers."
+	desc = "An advanced stun revolver with the capacity to shoot both disablers and lasers."
 	icon = 'sunset/icons/obj/guns/energy.dmi'
+	cell_type = /obj/item/stock_parts/cell{charge = 750; maxcharge = 750}
 	icon_state = "bsgun"
 	item_state = "gun"
 	force = 7
-	ammo_type = list(/obj/item/ammo_casing/energy/electrode/hos, /obj/item/ammo_casing/energy/laser/hos)
+	ammo_type = list(/obj/item/ammo_casing/energy/disabler, /obj/item/ammo_casing/energy/laser/hos)
```
Comment (alexkar598) [inline sunset/code/modules/projectiles/gun/blueshield.dm]: OwO you mad?

**Comment by Quantum-M** [comment]
I think the morgue access should be removed, along with ACCESS_HEADS, if it access the Head's lockers and offices, to prevent any Blueshield from rummaging through the stuff.

**Comment by alexkar598** [comment]
Access head is bridge and stuff like that    . And for morgue access,its not like anything important is in there so it should be done

**Comment by miniusAreas** [comment]
Anywhere a Head Of Staff should be going, the BS should have access. Save for things like Genpop, Armory, Virology, Mining, Genetics, Bar/kitchen, Xenobio, Engine, etc, They need to be able to rush in to save a Head of staff, and yelling at a AI or another person takes away time that could save a Head from death or being kidnapped.

**Comment by azzzertyy** [comment]
Bluespace doesnt have bridge accesss. Fix. They have access to the captains locker... but not bridge access.

**Comment by alexkar598** [comment]
Il see for bridge access when I can. As for increasing his access,il probably end up giving some more departmental access for mobility but not a lot. Perhaps xenobio,engineering,atmos and cargo bay

**Comment by alexkar598** [comment]
Gave the poor dude atmos access due to being a big chunk of engineering,also bridge access is there but i standarised the access for high and low pop

**Comment by SmokingKilz** [comment]
Main issue I have with the ballistic removal is how useless it makes a Blueshield against anything energy resistant which can be a lot of things. That being said; I don't mind seeing him only get an e-revolver as long its not just a really shitty re-sprited e-gun. 

Potentially if we bring down the shots to 6 and beef up the damage heavily, that could be an interesting concept to play around with. Possibly even making it so instead of recharging the whole gun, you get power packs which you could individually swap in and out. Would make the gun a lot more unique and definitely something really interesting to consider.

**Comment by ghost** [comment]
didn't we learn the hard way some lessons about the blueshield on oracle

**Comment by alexkar598** [comment]
@SmokingKilz he still has the enforcer

**Comment by alexkar598** [comment]
@SmokingKilz it isint that shittier compared to an egun,its 15 disabler insteed of 20 disabler and 7 lethal insteed of 10 lethal and as for the briging shots down for more fire power that is exactly the opposite of what i want the blueshield to be,that is shoots alot but dosent do alot of trouble.

**Review by ghost — APPROVED** [review]

**Pull request merged**